spec/requests/users_spec.rb

```diff
@@ -2,6 +2,29 @@
 
 RSpec.describe 'Users', type: :request do
   describe 'GET /index' do
-    pending "add some examples (or delete) #{__FILE__}"
+    before(:example) { get '/users' }
+    it 'returns http success' do
+      expect(response).to have_http_status(:success)
+    end
+    it 'renders users index view' do
+      expect(response).to render_template(:index)
+    end
+    it 'shows content in the view' do
+      expect(response.body).to include('Here is a list of users')
+    end
+  end
+
+  describe 'GET /show' do
+    before(:example) { get users_path }
+
+    it 'responds with the code 400' do
+      expect(response).to have_http_status(:success)
+    end
+    it 'renders show view' do
+      expect(response).to render_template(locals: 'users/show')
+    end
+    it 'shows contents in the view' do
+      expect(response.body).to include('Here is a list of users')
+    end
```

Comment:
- [ ] The same problem referenced above ☝️ occurs here as you can see below:
![image](https://user-images.githubusercontent.com/46839548/186209797-0e81701a-15c1-4c1b-8f85-8864bde5eef0.png)